RxNimble.podspec

```diff
@@ -20,8 +20,8 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |ss|
     ss.source_files  = "Source/Core/"
-	ss.dependency "Nimble", "~> 7.0"
-	ss.dependency "RxSwift", "~> 4.2"
+	ss.dependency "Nimble", "~> 8.0"
+	ss.dependency "RxSwift", "~> 5.0"
```

Comment:
I understand that this update is something that makes sense as part of the "breaking changes package", but I'm wondering, if we're not changing anything else in the project that _requires_ this update, should we force RxNimble users to update those libraries when they try to update RxNimble?

On the other hand, if this is considered a good practice when updating the major version because sooner or later we're going to use RxSwift 5+ and Nimble 8+ stuff, then I'm good :+1: please let me know your thoughts!

Reply:
I think that as long as we're pushing up a major release number, it's best to be on the latest major versions of the other dependencies, because otherwise adding deprecations is harder.

Reply:
Thanks, agreed